stock_inventory_verification_request/models/stock_slot_verification_request.py

```diff
@@ -0,0 +1,136 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class SlotVerificationRequest(models.Model):
+    _name = 'stock.slot.verification.request'
+    _inherit = 'mail.thread'
+
+    @api.model
+    def create(self, vals):
+        vals['name'] = self.env['ir.sequence'].next_by_code(
```

Comment:
Maybe we should add a check to avoid forcing the name if someone calls the `create` method with a name already defined ?